packages/experiments/src/components/FloatingPicker/PeoplePicker/PeoplePickerItems/SuggestionItemDefault.tsx

```diff
@@ -12,7 +12,7 @@ export const SuggestionItemNormal: (persona: IPersonaProps, suggestionProps?: IB
       <div className={ css('ms-PeoplePicker-personaContent', stylesImport.peoplePickerPersonaContent) }>
         <Persona
           presence={ personaProps.presence !== undefined ? personaProps.presence : PersonaPresence.none }
-          size={ PersonaSize.size24 }
+          size={ PersonaSize.small }
```

Comment:
small is deprecated. Any reason to change it from size24? (in a couple places here.)

Reply:
Whoops, didn't realize it was deprecated. Will change to the numeric sizing.